Sources/Vitamin/Components/Progressbar/VitaminProgressbar.swift

```diff
@@ -354,6 +366,7 @@ extension VitaminProgressbar {
             case .image:
                 self.percentageLabel.isHidden = true
                 self.centerImage.isHidden = false
+                accessibilityValue = nil
```

Comment:
I think the value should always be set, with the exception when `progressType` is `.indeterminate`
```suggestion
```